src/search-schema/data/definitions/enums/assay_types.yaml

```diff
@@ -2,18 +2,22 @@ AF:
     description: Autofluorescence Microscopy
     alt-names: []
     primary: true
+    contains-pii: false
     vitessce-hints: []
  
-# AF_pyramid:
-#     description: Autofluorescence Microscopy [Image Pyramid]
-#     alt-names: [["AF", "Image Pyramid"], ["Image Pyramid", "AF"]]
-#     primary: false
-#     vitessce-hints: ['pyramid']
+AF_pyramid:
+    description: Autofluorescence Microscopy [Image Pyramid]
+    alt-names: [["AF", "Image Pyramid"], ["Image Pyramid", "AF"]]
+    primary: false
+    vis-only: true
+    contains-pii: false
+    vitessce-hints: ['pyramid']
 
 ATACseq-bulk:
     description: Bulk ATAC-seq
     alt-names: []
     primary: true
+    contains-pii: true
     vitessce-hints: []
 
 bulk_atacseq:
```

Comment:
This is a bad key name, as it represents derived data but it is nearly identical to the raw that that DOES contain PII. The key should have some indication that this is derived data after processing.